packages/block-library/src/post-featured-image/edit.js

```diff
@@ -226,6 +236,14 @@ export default function PostFeaturedImageEdit( {
 		setTemporaryURL();
 	};
 
+	const imperativeFocalPointPreview = ( value ) => {
+		if ( ! imageElement.current ) {
+			return;
+		}
+		// eslint-disable-next-line react-compiler/react-compiler
+		imageElement.current.style.objectPosition = mediaPosition( value );
```

Comment:
We won't need the ESLint ignore if we use `setProperty()`:
```suggestion
		imageElement.current.style.setProperty(
			'objectPosition',
			mediaPosition( value )
		);
```